Analyses/src/PointerCheck_module.cc

```diff
@@ -38,7 +40,54 @@ namespace mu2e {
   class PointerCheck : public art::EDAnalyzer {
 
   public:
-    explicit PointerCheck(fhicl::ParameterSet const& pset);
+
+    struct Config {
+      using Name=fhicl::Name;
+      using Comment=fhicl::Comment;
+
+      fhicl::OptionalAtom<bool> skipDereference{Name("skipDereference"), 
+	  Comment("skip dereferencing pointers")};
+      fhicl::OptionalAtom<int> verbose{Name("verbose"), 
+	  Comment("verbose flag, 0 to 10")};
```

Comment:
It would make more sense to have Atom<bool> with a default value rather than OptionalAtom<bool> that requires more lines of code.   Also,  in configuration description art prints defaulted values for Atoms, but does not know what Optional entries imply.    Optional constructs are necessary when the default values is not static but must be determined at run time (e.g. from geometry), or when a default does not make sense (e.g. for a complicated piece of subconfig for a helper class that does not get used).